pkg/api/v1beta3/dynakube/kspm.go

```diff
@@ -0,0 +1,57 @@
+package dynakube
```

Comment:
should we have it in a separate package, similar to https://github.com/Dynatrace/dynatrace-operator/pull/3895?

Reply:
valid point - let us merge your PR first and I will resolve

Reply:
(hopefully correctly) done in https://github.com/Dynatrace/dynatrace-operator/pull/3906/commits/997c5369ec80a21be073ceb450105adf9053152f

Reply:
🧑‍🍳 💋